### PR TITLE
Rework console output to use logging instead of p()

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/207)

### CODE REVIEW

File: downloader/src/main/kotlin/Main.kt [hunk 1]
```diff
@@ -204,60 +201,82 @@ object Main {
...
+        } else {
+            println("Using VCS URL '${target.vcsProcessed.url}'.")
         }
 
         if (target.vcsProcessed.revision.isBlank()) {
-            p("WARNING: No VCS revision provided, downloaded source code does likely not match revision " +
-                    target.version)
+            log.warn {
+                "No VCS revision provided, downloaded source code will probably not match package version " +
+                        "${target.version}."
```
Comment (mnonnenmacher) [inline downloader/src/main/kotlin/Main.kt]: No need to quote this.

Comment (sschuberth) [reply downloader/src/main/kotlin/Main.kt]: What about the trailing `.`? 😉 I *could* write

    + target.version + "."

if you find that more readable.

Comment (mnonnenmacher) [reply downloader/src/main/kotlin/Main.kt]: No, I missed the dot.

**Comment by sschuberth** [comment]
Tests depends on the fix in https://github.com/heremaps/oss-review-toolkit/pull/210 to succeed.

**Comment by mnonnenmacher** [comment]
@sschuberth I have rebased the branch for you.

**Comment by sschuberth** [comment]
@mnonnenmacher, the Travis error is a bogus one, and Travis is still down. So I'll merge despite Travis once your approval is there.

**Pull request merged**